src/commands/copy.json

```diff
@@ -82,7 +82,7 @@
                     "const": 1
                 },
                 {
-                    "description": "Source was not copied.",
+                    "description": "Source was not copied when the destination key already exists",
```

Comment:
Is this description actually used anywhere?

Reply:
The words come from our webpage https://valkey.io/commands/move/ and https://valkey.io/commands/copy/,  it makes user easier to understand

Reply:
The reply text on the website comes from the doc repo `resp2_replies.json` and `resp3_replie.json`.